### PR TITLE
refactor: remove project token, unused since 0.19.0

### PR DESCRIPTION
**Pull request by tgriesser**
### Additional details

Noticed this while going through the `project-base` code, figured we could rip this out since it's no longer supported in the dashboard.

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/desktop-gui/src/runs/runs-list.jsx [hunk 1]
```diff
@@ -41,11 +41,11 @@ class RunsList extends Component {
   componentDidMount () {
     this._pingApiServer()
     this._handlePolling()
-    this._getKey()
+    this._getRecordKeys()
```
Comment (Bkucera) [inline packages/desktop-gui/src/runs/runs-list.jsx:44]: this is an unrelated rename correct?

**Comment by Ponthius** [comment]
Hi is there any way I can get free subscribers for my YouTube channel
called "Ponthius Gamer" please help me out thank you.

On Jun 21, 2021 23:19, "Tim Griesser" ***@***.***> wrote:

Additional details

Noticed this while going through the project-base code, figured we could
rip this out since it's no longer supported in the dashboard.
PR Tasks

   - Have tests been added/updated?
   - Has the original issue or this PR been tagged with a release in
   ZenHub?
   - Has a PR for user-facing changes been opened in cypress-documentation
   <https://github.com/cypress-io/cypress-documentation>?
   - Have API changes been updated in the type definitions
   <https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts>
   ?
   - Have new configuration options been added to the cypress.schema.json
   <https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json>
   ?

------------------------------
You can view, comment on, or merge this pull request online at:

  https://github.com/cypress-io/cypress/pull/17039
Commit Summary

   - chore: remove project token route & related fns, unsupported since
   0.19.0

File Changes

   - *M* packages/desktop-gui/src/runs/runs-list.jsx
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-4a6fb8584478050f6064ddd7054ebf98e3b51c1b855aafc678b4923dead6b09a>
   (6)
   - *M* packages/server/lib/api.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-1d59c84a881e220507b9f0d77df9b4287221596d218888edb3d4cb3d7e40e030>
   (24)
   - *M* packages/server/lib/cypress.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-fd8260e3de42c8cd3e678bac66b13db87507e88b3b4b3919d7463645bc345b28>
   (22)
   - *M* packages/server/lib/project-base.ts
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-a062d0b0db7bfd6f4a0d1785e9d83ee00e169d21dbb2cdcf7107ed0fab857e6a>
   (28)
   - *M* packages/server/lib/util/args.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-d58611ce5d9b3d8c6efd8eeac201ae432914f95fd367d1a62f4af0ae99a6ed77>
   (4)
   - *M* packages/server/lib/util/routes.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-c8386bea0120f7411c882a6ecb813f87883d65755d4aa679226cfb440b78a3aa>
   (1)
   - *M* packages/server/test/integration/cypress_spec.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-0416ec0277f3a43b360ddb7f3b56280f1ab1851592bc36e90fcffe74ee571ee8>
   (140)
   - *M* packages/server/test/unit/args_spec.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-11481b4408ea5a3b071cffd50825ffb1663036ff30d909d721de67a0151f4cae>
   (4)
   - *M* packages/server/test/unit/project_spec.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-65758f8a110a2725f2ca0bb53d991bc3a3e74a74233b522b83e6be3c7910e697>
   (58)
   - *M* packages/server/test/unit/routes_util_spec.js
   <https://github.com/cypress-io/cypress/pull/17039/files#diff-1ec41d087e289d00f2bc58861314e44c4631317bf502d79ccfea97d97604b287>
   (4)

Patch Links:

   - https://github.com/cypress-io/cypress/pull/17039.patch
   - https://github.com/cypress-io/cypress/pull/17039.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/cypress-io/cypress/pull/17039>, or unsubscribe

File: packages/desktop-gui/src/runs/runs-list.jsx (hunk 1 above)
Comment (tgriesser) [reply packages/desktop-gui/src/runs/runs-list.jsx:44]: @Bkucera yeah somewhat, basically just being more explicit about what it's doing.

**Comment by jennifer-shehane** [comment]
@tgriesser Would this be technically breaking anything for 0.19.0 users? Will probably lump in with 8.0 if so which is upcoming really soon.

**Comment by tgriesser** [comment]
@jennifer-shehane nah, and I would be shocked if there were any active users of Cypress 0.19.0 🤣

**Review by chrisbreiding — APPROVED** [review]
LGTM 👍 Love to see obsolete code cleaned up
